docs/syntax/metrics.md

```diff
@@ -7,117 +7,43 @@ permalink: /docs/syntax/metrics/
 
 # Metrics format
 
-All Origami web services are required to expose multiple `/__metrics` endpoints (one for each version, and one at the root) to publish metrics on the current state of the system.  The response give at this URL must be a JSON document conforming to the following format.
+All Origami web services are encouraged to emit metrics over TCP or UDP to the FT graphite service.  Metrics keys are heirarchical, delimited by dots, and the first two levels are standardised:
 
-## Format
+	{appName}.{envName}.*
+
+Your `appName` *must* be common name of your application, which *should* be the name of your code repository, the CMDB name and the first token of the service's hostname.  The `appName` *must* distinguish the application from any other services run by the FT.
```

Comment:
How do we verify this string in the 'secure gateway' proxy?  The proxy needs to hold a list of valid names so that it can pass them through to the real Carbon daemon, or drop them and terminate the connection if it is an invalid root namespace.